### PR TITLE
Ml pipeline

### PR DESCRIPTION
**Pull request by lyriccoder**
fix https://github.com/yegor256/aibolit/issues/237

### CODE REVIEW

**Comment by lyriccoder** [comment]
Do not merge, do not review. I will talk about it with @acheshkov

**Comment by lyriccoder** [comment]
@0crat out

**Comment by acheshkov** [comment]
@0crat out

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/aibolit/issues/238#issuecomment-617881521))

@lyriccoder Job `gh:yegor256/aibolit#238` is not assigned, can't get performer

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/aibolit/issues/238#issuecomment-617882339))

@acheshkov Job `gh:yegor256/aibolit#238` is not assigned, can't get performer

File: aibolit/__main__.py [hunk 1]
```diff
@@ -24,19 +24,16 @@
...
-from pathlib import Path
 
 import torch
 
 from aibolit import __version__
 from aibolit.metrics.entropy.entropy import Entropy
 from aibolit.metrics.ncss.ncss import NCSSMetric
 from aibolit.metrics.spaces.SpaceCounter import IndentationCounter
-from aibolit.model.model import Net  # type: ignore
+from aibolit.ml_pipeline.ml_pipline import *
```
Comment (acheshkov) [inline aibolit/__main__.py:36]: mispell `ml_pipl___ine `

**Comment by acheshkov** [comment]
@rultor merge

**Comment by acheshkov** [comment]
@lyriccoder please, fix formatting issues

**Comment by lyriccoder** [comment]
Fixed

**Comment by acheshkov** [comment]
@rultor merge

**Comment by acheshkov** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/aibolit#238` is not assigned, can't get performer

**Pull request merged**